odoo/addons/base/models/ir_actions.py

```diff
@@ -812,12 +812,21 @@ def _traverse_path(self):
 
     def _get_relation_chain(self, searched_field_name):
         self.ensure_one()
-        if not searched_field_name or not searched_field_name in self._fields or not self[searched_field_name]:
+        if (
+            not searched_field_name
+            or searched_field_name not in self._fields
+            or not self[searched_field_name]
+            or not self.model_id
+            or not self.model_id.model
+        ):
             return [], ""
         path = self[searched_field_name].split('.')
         if not path:
             return [], ""
-        model = self.env[self.model_id.model]
+        try:
+            model = self.env[self.model_id.model]
+        except KeyError:
+            return [], ""
```

Comment:
```suggestion
        model = self.env[self.model_id.model]
```
No need try block; above condition solves the issue.

Reply:
changes are updated , please review them 

Thanks :)